vendor/github.com/longhorn/backupstore/deltablock.go

```diff
@@ -1100,6 +1101,7 @@ func getLatestBackup(backup *Backup, lastBackup *Backup) error {
 	}
 
 	if backupTime.After(lastBackupTime) {
+		// FIXME - go lint points out that this copies a potentially locked sync.mutex
```

Comment:
is this comment intended 
```
// FIXME - go lint points out that this copies a potentially locked sync.mutex
```

Reply:
Yes.  It's a lint error that I got while running :GoLint in my IDE, and it looks real, but not to address in this PR.